### PR TITLE
Add 'messaging brain' tracking

### PR DESCRIPTION
**Pull request by remybach**
Goes along with https://github.com/Financial-Times/o-tracking/pull/162

 🐿 v2.5.13

### CODE REVIEW

**Review by geek-caroline — APPROVED** [review]

**Review by geek-caroline — APPROVED** [review]
I shall add the same 'beta' release into my gift article branch

**Comment by remybach** [comment]
Yeah, I figured it was safer to test this in our own environments first!

File: main.js [hunk 1]
```diff
@@ -32,6 +41,15 @@ export default class DesktopAppBanner {
 		if (!this.emailButton.disabled) {
 			this.emailButton.disabled = 'disabled';
 
+			document.body.dispatchEvent(new CustomEvent('oTracking.event', {
+				detail: {
+					category: 'form',
+					action: 'submit',
+					messaging: 'desktop-app-banner-form-submit'
```
Comment (tom-parker) [inline main.js:48]: Was the idea not to have the messaging attribute consistent across all of the events for a given component? e.g 'desktop-app-banner' will probably be sufficient everywhere?

Comment (remybach) [reply main.js:48]: Yeah, that makes more sense... I'll update it!

**Pull request merged**